uptream: yes (master)

From 29bf8d8528ec694f65c8fae310adac996322cc74 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <jan.steffens@ltnglobal.com>
Date: Sat, 9 May 2020 19:59:46 +0200
Subject: [PATCH] vulkan: Drop use of VK_RESULT_BEGIN_RANGE

This was removed in Vulkan 1.2.140.

> Shortly after 2020-04-24, we will be removing the automatically
> generated `VK_*_BEGIN_RANGE`, `VK_*_END_RANGE`, and `VK_*_RANGE_SIZE`
> tokens from the Vulkan headers. These tokens are currently defined for
> some enumerated types, but are explicitly not part of the Vulkan API.
> They existed only to support some Vulkan implementation internals,
> which no longer require them. We will be accepting comments on this
> topic in [#1230], but we strongly suggest any external projects using
> these tokens immediately migrate away from them.

[#1230]: https://github.com/KhronosGroup/Vulkan-Docs/issues/1230
---
 ext/vulkan/vkerror.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/ext/vulkan/vkerror.c b/ext/vulkan/vkerror.c
index 3fec27e4d..c91589d9b 100644
--- ext/vulkan/vkerror.c
+++ ext/vulkan/vkerror.c
@@ -27,7 +27,7 @@
 #include "vkerror.h"
 
 /* *INDENT-OFF* */
-static const struct 
+static const struct
 {
   VkResult result;
   const char *str;
@@ -63,8 +63,6 @@ _vk_result_to_string (VkResult result)
 
   if (result >= 0)
     return NULL;
-  if (result < VK_RESULT_BEGIN_RANGE)
-    return "Unknown Error";
 
   for (i = 0; i < G_N_ELEMENTS (vk_result_string_map); i++) {
     if (result == vk_result_string_map[i].result)
-- 
2.26.2

